ci/Dockerfile

```diff
@@ -8,37 +8,21 @@ FROM ${DOCKER_IMAGE_ROOT} as base
 ENV DEBIAN_FRONTEND=noninteractive
 
 RUN apt update --fix-missing && \
-    apt -y install python3.8 python3.8-venv python3-pip \
-    binutils git wget \
-    scons build-essential pkg-config \
-    unzip texlive-latex-base graphviz nano vim \
-    libgtk2.0-0 libgtk-3-0 libxss1 libasound2 plantuml && \
```

Comment:
we need to inform the users that if they would like to work with graphical interface inside the docker, they should add:
libgtk2.0-0 libgtk-3-0 libxss1 libasound2 to the docker env.
In readme? Documentation?

Reply:
Not sure, for now we don't mention graphical interface anywhere at all, so it's not like we're removing a documented feature. Documentation is pretty much an unrelated task here.